src/types/common.d.ts

```diff
@@ -0,0 +1,22 @@
+export interface MusicAccidental {
+  note: number;
```

Comment:
Why is note used instead of root?

Reply:
This is an inconsistency, but it does not belong to this PR. Let's fix it after we complete the migration